### PR TITLE
docs: disable smartypants

### PR DESCRIPTION
**Pull request by enkore**

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
what's the point?

File: docs/conf.py [hunk 1]
```diff
@@ -148,7 +148,9 @@
 
 # If true, SmartyPants will be used to convert quotes and dashes to
 # typographically correct entities.
-#html_use_smartypants = True
+#
+# This is disabled to avoid mangling --options-that-appear-in-texts.
```
Comment (enkore) [inline docs/conf.py]: .

Comment (ThomasWaldmann) [inline docs/conf.py]: is this a problem if they are written like this:
```
bla bla ``--option`` bla bla
```
?

Comment (enkore) [inline docs/conf.py]: There are a *lot* of occurrences that doesn't use literals for them. There's also the issue of rST syntax in --help output (and "borg help XYZ").

Comment (ThomasWaldmann) [inline docs/conf.py]: ah, ok.

**Pull request merged**